lib/driver.js

```diff
@@ -986,6 +986,7 @@ class XCUITestDriver extends BaseDriver {
       shouldTerminateApp: this.opts.shouldTerminateApp ?? true,
       forceAppLaunch: this.opts.forceAppLaunch ?? true,
       useNativeCachingStrategy: this.opts.useNativeCachingStrategy ?? true,
+      forceTurnOnSoftwareKeyboard: this.opts.forceTurnOnSoftwareKeyboard ?? true,
```

Comment:
would the default still work as expected is connectHardwareKeyboard is set to false?

Reply:
Yes, this caps works when creating a new session. the hardware keyboard is when preparing a simulator instance. This capability set the software keyboard when I tested on my local.